ci/asv.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "inside $0"
+
+source activate pandas
+
+RET=0
+
+if [ "$ASV" ]; then
+    echo "Check for failed asv benchmarks"
+
+    pip install git+https://github.com/spacetelescope/asv
+
+    cd asv_bench
+
+    time asv run --quick | grep "failed"
+
+    if [ $? = "0" ]; then
+        RET=1
+    fi
+    echo "Check for failed asv benchmarks DONE"
+fi
+
```

Comment:
can you add an else like we do in lint. so that we get a message if this is NOT run.